src/cli/mod.rs

```diff
@@ -201,6 +203,39 @@ pub enum Command {
         migration: AdhocMigration,
     },
 
+    /// Delete a Workers script from an account.
+    #[structopt(name = "delete")]
+    Delete {
+        /// Disables interactive mode
+        #[structopt(long = "no-interactive", short = "n")]
+        no_interactive: bool,
+        /// Allows to delete script without confirmation step. WARNING: It will delete the Worker script and any bound Durable Object namespace.
+        #[structopt(
+            long,
+            short = "f",
+            requires = "no-interactive",
+            requires = "account-id",
+            requires = "script-id"
+        )]
+        force: bool,
+        /// Account ID to delete the script from. Required if interactive mode is disabled.
+        #[structopt(
+            long = "account-id",
+            short = "aid",
+            requires = "no-interactive",
+            requires = "script-id"
+        )]
+        account_id: Option<String>,
```

Comment:
Even in non-interactive mode, we we use the default `account_id`? (from the credentials) If there are more than 1 options, then we can require it.

Reply:
Hmm, wrangler uses the account_id from `wrangler.toml` everywhere else, right? Why have it as a command line parameter here instead?

Reply:
The idea I had in mind was to let users run `wrangler delete` from any directory even if they don't have a Workers script on their machine.

Reply:
> The idea I had in mind was to let users run wrangler delete from any directory even if they don't have a wrangler.toml.

I can see this being a nice feature, but since we don't support deploying a project w/o a `wrangler.toml`, then it's probably safer & more consistent to maintain the same restriction when deleting. 

It is not uncommon for other deployment systems to _reference_ code artifacts to deploy (or tear down) without having the artifact / source locally though. So I do see this as generally a good idea, but maybe not for now for wrangler as it stands.

Reply:
> > The idea I had in mind was to let users run wrangler delete from any directory even if they don't have a wrangler.toml.
> 
> I can see this being a nice feature, but since we don't support deploying a project w/o a `wrangler.toml`, then it's probably safer & more consistent to maintain the same restriction when deleting.
> 
> It is not uncommon for other deployment systems to _reference_ code artifacts to deploy (or tear down) without having the artifact / source locally though. So I do see this as generally a good idea, but maybe not for now for wrangler as it stands.

I see, that makes a lot of sense. Thanks everyone for the feedback!